tactics/ind_tables.ml

```diff
@@ -107,7 +103,7 @@ let define_individual_scheme_base kind suff f mode idopt (mind,i as ind) =
     | Some id -> id
     | None -> add_suffix mib.mind_packets.(i).mind_typename suff in
   let role = Evd.Schema (ind, kind) in
-  let const, neff = define mode role id c (Declareops.inductive_is_polymorphic mib) ctx in
+  let const, neff = define (mode == InternalTacticRequest) role id c (Declareops.inductive_is_polymorphic mib) ctx in
```

Comment:
why the use of `==` ? you could also do `let internal = .... in ` as to have the code a bit more clearer.

Reply:
OK.